location/templates/location.html

```diff
@@ -23,4 +23,18 @@
 {% block content %}
 <h1>{{object.address}}, {{object.city}}, {{object.state}}, {{object.zipcode}}</h1>
 {% leaflet_map "leaflet_map_main" %}
+
+<form action="{% url 'favorite' object.id %}" method="POST">
+{% csrf_token %}
+<button type="submit" class="btn secondaryAction btn-outline-primary">Favorite</button>
+</form>
```

Comment:
I think it would be good to also have a button to remove a location from your favorites if it has already been added.

Reply:
Yeah I was thinking about adding a button for removing a location from favorites which should be relatively straightforward. But I think it might not be the most elegant way of doing it. Perhaps we should make it like a favorites icon which is yellow if the apartment is favorited and white otherwise, so that we don't have to redo the work later. Let me know what you think and if it should be done as part of this PR?

![image](https://user-images.githubusercontent.com/40494713/68182692-1aa1be00-ff68-11e9-9ddd-bd400b9d0218.png)

Reply:
great idea!

Reply:
I think that's a great idea! If you're able to it would be great to get that as part of this PR.

Reply:
added unfavorite functionality and used icons instead of buttons